src/controller/data_model/zap_upgrades.yaml

```diff
@@ -0,0 +1,8 @@
+upgrade_results:
```

Comment:
this file seems repeated in various places. Is it needed? Could we have it only once?

Reply:
I can put the upgrade results of all .zap files in one file. Where would you like me to put this?

Reply:
It is created per application upgrade but the convert script can be changed such that all the outputs just go into one file and we need not have this per application.